.github/workflows/nucliadb_dataset.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   pre-checks:
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
I wonder if it's safe to just check the lowest and highest versions we support? To alleviate a bit CI.
In this case: removing 3.10 from the list

Reply:
Maybe. Honestly, I think running the pre-checks across multiple versions does very little, except maybe the oldest version to check for unsupported old syntax. Ideally we would also run the actual tests across all versions, but that is very slow :sweat_smile: